File: December 2021/Day 4/Que5.java
// Que LINK: https://leetcode.com/problems/check-if-the-sentence-is-pangram/

class Solution {
    public boolean checkIfPangram(String sentence) {
        Set<Character> s = new HashSet<>();
        for (int i = 0; i < sentence.length(); ++i)
            s.add(sentence.charAt(i));
        return s.size() == 26;
    }
}

File: December 2021/Day 4/Que5.py
# Que LINK: https://leetcode.com/problems/check-if-the-sentence-is-pangram/

# Method 1:
class Solution:
    def checkIfPangram(self, sentence: str) -> bool:
        abc = 'abcdefghijklmnopqrstuvwxyz'
        for i in sentence:
            abc = abc.replace(i,'') 
        return (abc=='')

# Method 2:
class Solution:
    def checkIfPangram(self, sentence: str) -> bool:
        return len(set(sentence)) == 26